cmd/uhc/cluster/describe/describe.go

```diff
@@ -29,23 +30,45 @@ import (
 
 var args struct {
 	debug bool
+	out   string
+	short bool
 }
 
 var Cmd = &cobra.Command{
-	Use:   "describe CLUSTERID",
+	Use:   "describe CLUSTERID [--out] [--short]",
 	Short: "Decribe a cluster",
 	Long:  "Get info about a cluster identified by its cluster ID",
 	Run:   run,
 }
 
+// Check error
+func check(e error) {
+	if e != nil {
+		panic(e)
+	}
+}
```

Comment:
I don't like very much the use of _panic_, it think it is better to treat each error independently, and generate reasonable error messages. So please don't use this `check` function, check the error in place:

```go
err = doSomethingThatMayFail(...)
if err != nil {
        fmt.Fprintf(os.Stderr, "Something that may fail actually failed: %v", err)
        os.Exit(1)
}
```